### DISCUSSION TITLE
AttributeError: module 'pyvista.core.objects' has no attribute 'Texture'

### DISCUSSION THREAD

**Discussion by aniketgupta2009** [cgre-aachen/gemgis#281] [Q&A] [answered]
Hi,

I am trying to install Gemgis on the macOS M2 chip using the recommended procedure from the README file. The installation goes perfectly but while importing the Gemgis I am getting this error: "AttributeError: module 'pyvista.core.objects' has no attribute 'Texture'".

Please let me know how I can get rid of this error.

Thanking You.

<img width="1280" alt="gemgis" src="https://github.com/cgre-aachen/gemgis/assets/50705068/005aebe6-e365-423b-99b7-6c7c2bee68ce">

**Comment by AlexanderJuestel** [comment, marked as answer]
@aniketgupta2009 Thanks for reporting this. This will be fixed in GemGIS 1.0.10 which will be released soon!

**Comment by EdsonSR23** [comment]
Me too! Following the steps of installing the library, returns the same error
